### PR TITLE
Adding animation-duration mixin

### PR DESCRIPTION
**Pull request by SKempin**
Adding animation-duration mixin.

### CODE REVIEW

File: elements.less [hunk 1]
```diff
@@ -69,6 +69,17 @@
...
 
 
+/* Animation Mixins
+ * ============================================================ */
+.animation-duration(...) {
+  -webkit-animation-duration: @arguments;
+  -moz-animation-duration: @arguments;
+  -o-animation-duration: @arguments;
+  animation-duration: @arguments;
+}//.animation-duration
```
Comment (CITguy) [inline elements.less]: Please indent so that the colons align vertically.

File: elements.less [hunk 2]
```diff
@@ -69,6 +69,17 @@
...
 
 
+/* Animation Mixins
+ * ============================================================ */
+.animation-duration(...) {
+  -webkit-animation-duration: @arguments;
+	 -moz-animation-duration: @arguments;
+	   -o-animation-duration: @arguments;
+		  animation-duration: @arguments;
+}//.animation-duration
```
Comment (CITguy) [inline elements.less]: Better, but more like this...

```
.animation-duration(...) {
  -webkit-animation-duration: @arguments;
     -moz-animation-duration: @arguments;
       -o-animation-duration: @arguments;
          animation-duration: @arguments;
}//.animation-duration
```

Comment (CITguy) [inline elements.less]: It appears that your update may be mixing tabs and spaces.

Comment (SKempin) [inline elements.less]: Have replaced with your snippet. Can you clarify what settings should be used? Thanks

**Comment by CITguy** [comment]
@SKempin: `v1.2.0` now available. Thanks for your contribution!

**Pull request merged**